{{ cookiecutter.project_shortname }}/setup.cfg

```diff
@@ -11,8 +63,16 @@ universal = 1
 [pydocstyle]
 add_ignore = D401
 
-[pycodestyle]
-exclude = docs/conf.py
+[isort]
+profile=black
+
+[check-manifest]
```

Comment:
it should be changed the check_manifest line in run-tests.sh too
from `python -m check_manifest --ignore ".*-requirements.txt"` to `python -m check_manifest`